planner/core/task.go

```diff
@@ -46,6 +46,18 @@ var (
 	_ task = &mppTask{}
 )
 
+const (
```

Comment:
Why use `floate64` not integer?

Reply:
Though I thought integer is more proper, the estimate rows are represented as float64 in the optimizer. 🤔

Reply:
changed it to integer, PTAL again.